src/components/Page/components/Header/Header.tsx

```diff
@@ -184,16 +184,17 @@ export default class Header extends React.PureComponent<Props, State> {
     const actionGroupsMarkup =
       actionGroups.length > 0
         ? actionGroups.map(({title, icon, actions, details}) => (
-            <ActionGroup
-              key={title}
-              title={title}
-              icon={icon}
-              actions={actions}
-              details={details}
-              onOpen={this.handleActionGroupOpen}
-              onClose={this.handleActionGroupClose}
-              active={title === openActionGroup}
-            />
+            <div className={styles.IndividualAction} key={title}>
```

Comment:
Should this have a more unique key similar to the one given to secondary actions on line 270?

```suggestion
            <div
              className={styles.IndividualAction}
              key={`ActionGroup-${title}-${index}`}
            >
```